demo/views.py

```diff
@@ -19,7 +26,7 @@ def search(request):
         search_picks = query.editors_picks.all()
     else:
         search_results = Page.objects.none()
-        search_picks = EditorsPick.objects.none()
+        search_picks = SearchPromotion.objects.none()
```

Comment:
I don't think so - the wagtaildemo site front-end still refers to them as Editor's Picks (which is a sensible enough name when used within the context of a search interface - it's only confusing within wagtail admin, where 'editor' has a different meaning), so it makes sense for the template variable to match that.